ocs_ci/ocs/constants.py

```diff
@@ -644,3 +644,14 @@
 
 # gather bootstrap
 GATHER_BOOTSTRAP_PATTERN = 'openshift-install gather bootstrap --help'
+
+# local storage
+LOCAL_STORAGE_OPERATOR = os.path.join(
+    TEMPLATE_DEPLOYMENT_DIR, "local-storage-operator.yaml"
+)
+LOCAL_VOLUME_YAML = os.path.join(
+    TEMPLATE_DEPLOYMENT_DIR, "local-volume.yaml"
+)
+LOCAL_STORAGE_CLUSTER_YAML = os.path.join(
+    TEMPLATE_DEPLOYMENT_DIR, "local-storage-cluster.yaml"
```

Comment:
local-storage-cluster.yaml is missing in PR

Reply:
Good catch, I was originally using this file but removed it in favor of updating the data we normally pull for storage cluster. I removed this constant in the latest update.